src/PowerShell/Help/Microsoft.WinGet.Client/Get-WinGetUserSettings.md

```diff
@@ -0,0 +1,58 @@
+﻿---
+external help file: Microsoft.WinGet.Client.Cmdlets.dll-Help.xml
+Module Name: Microsoft.WinGet.Client
+ms.date: 08/01/2024
+online version:
+schema: 2.0.0
+title: Get-WinGetUserSettings
+---
+
+# Get-WinGetUserSettings
+
+## SYNOPSIS
+Gets user settings for WinGet.
+
+## SYNTAX
+
+```
+Get-WinGetUserSettings [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+This command displays the WinGet settings for the current user. The settings are stored in
+`$env:LOCALAPPDATA\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\LocalState\settings.json`.
+This file only exists if you have change a user setting using the `Set-WinGetUserSettings` command.
```

Comment:
That's not the only way settings can be changed. Maybe "...if you have changed a user setting, for example, using..."?